api/backend/ecs/load_balancer_manager.go

```diff
@@ -71,7 +71,14 @@ func (e *ECSLoadBalancerManager) GetLoadBalancer(loadBalancerID string) (*models
 		return nil, err
 	}
 
-	return e.populateModel(loadBalancer), nil
+	lbAttributes, err := e.ELB.DescribeLoadBalancerAttributes((*loadBalancer.LoadBalancerName))
+	if err != nil {
+		if ContainsErrCode(err, "LoadBalancerAttributeNotFound") {
+			return nil, errors.New(errors.LoadBalancerAttributeNotFound, err)
```

Comment:
we should only catch/handle errors if we can do something meaningful with it; this looks like we are just passing the error though here.